examples/streamlit/README.md

```diff
@@ -0,0 +1,13 @@
+# Streamlit deployment via `pynsist`
+
+This is an example on how to deploy a streamlit app using `pynsist`. The example also includes the required packages to run plotly inside the app.
+
+The extra wheels of the package `blinker`is required (can't be found on Pypi).
+
+[Here](https://stackoverflow.com/questions/69352179/package-streamlit-app-and-run-executable-on-windows/69621578#69621578) you can find additional information on why the `src/run_app.py` has the chosen structure.
+
+To properly run the app once installed, the environmental variable `WORKING_DIR` has to be set to the installation path on the Windows machine: `<installation-folder>/pkgs`.
+
```

Comment:
```suggestion
```

This is no longer needed, I think. :slightly_smiling_face:

Reply:
Right, I missed it. Removed.